### PR TITLE
Upgrade app to Rails 7.2

### PR DESCRIPTION
**Pull request by MuriloDalRi**
Upgrades Rails from 7.1 to 7.2

https://github.com/alphagov/govuk-infrastructure/issues/1444

### CODE REVIEW

**Comment by theseanything** [comment]
Do we need all the HTML error pages - as this app only serves a JSON API?

**Comment by MuriloDalRi** [comment]
> Do we need all the HTML error pages - as this app only serves a JSON API?

Probably not, but I just assumed since it already had some error pages that there was no harm in updating or adding a new one. Do you want me to remove them all?

**Comment by theseanything** [comment]
> > Do we need all the HTML error pages - as this app only serves a JSON API?
> 
> Probably not, but I just assumed since it already had some error pages that there was no harm in updating or adding a new one. Do you want me to remove them all?

Yah - be best not to add unused code to the repo

File: bin/setup [hunk 1]
```diff
@@ -1,8 +1,8 @@
 #!/usr/bin/env ruby
 require "fileutils"
 
-# path to your application root.
 APP_ROOT = File.expand_path("..", __dir__)
+APP_NAME = "content-performance-manager"
```
Comment (theseanything) [inline bin/setup:5]:
```suggestion
APP_NAME = "content-data-api"
```

**Review by theseanything — APPROVED** [review]

**Pull request merged**